backend-rust/src/indexer.rs

```diff
@@ -4229,7 +4276,13 @@ impl RestakeEarnings {
     async fn save(
         &self,
         tx: &mut sqlx::Transaction<'static, sqlx::Postgres>,
+        protocol_version: ProtocolVersion
```

Comment:
I cannot send the range along, as I am unaware what range the individual validations are using already.

Also a time may come where additional metadata may be needed.

I could send the metadata along, I am fine with this.